RecoLocalTracker/SiPixelClusterizer/plugins/alpaka/SiPixelRawToCluster.cc

```diff
@@ -152,6 +258,33 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       fedIds_ = cablingMap_->fedIds();
       cabling_ = cablingMap_->cablingTree();
       LogDebug("map version:") << cablingMap_->version();
+      const TrackerTopology* tTopo_ = &iSetup.getData(trackerTopologyToken_);
```

Comment:
The `recordWatcher_` monitors the `SiPixelFedCablingMapRcd`.
The `TrackerTopology` comes from the `TrackerTopologyRcd`.
Is there any guarantee that the IOV changes to the `TrackerTopologyRcd` are synchronised to those of the `SiPixelFedCablingMapRcd` ?

Reply:
From the framework point of view assuming two independent Records would have the same IOV would be considered as a bad practice.

Notably this use case of "create `morphingModuleHost` if either `SiPixelFedCablingMapRcd` or `TrackerTopologyRcd` has an IOV change and copy the data to the device(s)" is exactly what the (Alpaka) EventSetup system does. On that route a new Record would be created that depends on both `SiPixelFedCablingMapRcd` and `TrackerTopologyRcd`, and a new Alpaka ESProducer that produces the `morphingModulesHost` (and leaves the copy to devices to be done by the framework). The `SiPixelRawToCluster` would then get the `morphingModulesDevice` from the aforementioned new Record.